src/api/utils.py

```diff
@@ -0,0 +1,32 @@
+import json
+import os.path
+
+
+def load_json(file_path):
```

Comment:
One convention that we follow in our codebase is to end all variable names that are supposed to be paths with `p` for brevity. For eg: "file_path" would be "filep". Similarly any directory path ends with "dirp".